docs/docs/deploying-to-qovery.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Deploying to Qovery
+---
+
+## Introduction
+
+[Qovery](https://www.qovery.com) is a fully-managed Cloud platform where you can host your modern application (web, database, broker, storage).
+
+> Trusted by developers from great companies - Google, Red Hat, Elastic and more
+
+Qovery include the following:
+
+- Continuous, automatic builds & deploys from GitHub, Gitlab and Bitbucket.
+- [Custom domains](https://docs.qovery.com/services/network/dns).
+- Automatic TLS certificates through [Let's Encrypt](https://letsencrypt.org).
+- Microservices support.
+- Auto-scaling support.
+- Native HTTP/2 support.
+- [Isolated environment](https://docs.qovery.com/extending-qovery/branches-and-environments) per git branch.
+- Managed databases (PostgreSQL, MySQL, MongoDB, Elasticsearch, Redis, Memcached, Cassandra).
+- Managed brokers (RabbitMQ, Kafka).
+- Managed storage (S3).
+- Custom URL redirects and rewrites.
+
+## Assumptions
+
+- You have an account with GitHub, GitLab or Bitbucket.
+- You have completed the Quick Start or have a Gatsby website you are ready to deploy and share with the world.
+- You have a Qovery account. You can sign up at [start.qovery.com](https://start.qovery.com).
+
+## Deploy your application
+
+You can set up a Gatsby site on Qovery in five quick steps:
+
+**1/** Sign up
```

Comment:
Can you add a note about having to install the Qovery CLI before running the bash command ?